webapp-django/crashstats/crashstats/models.py

```diff
@@ -699,6 +699,7 @@ class RawCrash(SocorroMiddleware):
         "AdapterRendererIDs",
         "AdapterSubsysID",
         "AdapterVendorID",
+        "ApplicationBuildID",
```

Comment:
Adding this here makes the field show up in the Metadata tab of the crash report.